libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/treeMap/tests/PluggableTreemap.test.tsx

```diff
@@ -291,4 +302,32 @@ describe("PluggableTreemap", () => {
             ]);
         });
     });
+
+    describe("Drill Down", () => {
+        it.each([
+            [sourceInsightDef, Region, targetUri, intersection, expectedInsightDefRegion],
+            [sourceInsightDef, Department, targetUri, intersection, expectedInsightDefDepartment],
+        ])(
+            "should replace the drill down attribute and add intersection filters",
```

Comment:
The test name does not reflect the input, so if one fails, we would not know which.

Reply:
Fixed